jobboard/models.py

```diff
@@ -109,5 +111,13 @@ def get_jobs_by_recruiter_id(cls, recruiter: Recruiter):
         """
         return set(cls.objects.filter(recruiter__user_id=recruiter.user.id))
 
+    @classmethod
+    def get_jobs_by_major(cls, major):
```

Comment:
I think we might need to define the function as 
`def get_jobs_by_major(cls, *major)`
so the student will be able to filter jobs according to more than 1 major, and of course, change the functionality also, properly

Reply:
On it